yacc/main.c

```diff
@@ -100,9 +100,6 @@ char *nullable;
 #if !defined(HAS_MKSTEMP)
 extern char *mktemp(char *);
 #endif
-#ifdef _WIN32
-extern char *getenv(const char *);
-#endif
```

Comment:
Actually, I just looked at this in more detail and perhaps this fix isn't correct. 93c32dbd10d8b6f698aaabef46192a6d5370d268 eliminated the `NO_UNIX` define, but it was not correctly applied here. It was previously `#ifndef NO_UNIX` so it should have become `#ifndef _WIN32` not `#ifdef _WIN32`.

However, I'm not sure why you'd ever want this now - shouldn't the appropriate header be `#include`'d instead?

/cc @xavierleroy, @shindere